mautic/mautic
# Changes company links to action /view in reports

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | 3.3 
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no




#### Description:
Some links in the reports details were pointing toward a /edit action and we want all of them to point to a /view action.



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create a custom report and add some ID fields.
3. Check that the link on the ID point towards a 'view' action

OLD
![Screenshot 2021-03-09 at 10 24 16](https://user-images.githubusercontent.com/79137416/110450920-f3ab4d00-80c3-11eb-88a8-3976224e8f65.png)

NEW 
![Screenshot 2021-03-09 at 10 25 43](https://user-images.githubusercontent.com/79137416/110451010-0d4c9480-80c4-11eb-8a64-c5c53a23c4af.png)
